src/sage/groups/perm_gps/permgroup_named.py

```diff
@@ -262,20 +268,24 @@ def __init__(self, domain=None):
         # Note that we skip the call to the superclass initializer in order to
         # avoid infinite recursion since SymmetricGroup is called by
         # PermutationGroupElement
-        cat = Category.join([FinitePermutationGroups(), FiniteWeylGroups().Irreducible()])
+        cat = Category.join([FinitePermutationGroups(),
+                             FiniteWeylGroups().Irreducible()])
         super(PermutationGroup_generic, self).__init__(category=cat)
 
         self._domain = domain
-        self._deg = len(self._domain)
+        self._deg = n = len(self._domain)
```

Comment:
My only little quibble is can we just use `self._deg` instead of creating a new variable `n` ? Otherwise LGTM.